lib/common/delete-phases-common.js

```diff
@@ -59,9 +60,16 @@ exports.unDeployCloudFormationStack = function (serviceContext, serviceType) {
             }
         })
         .then(() => {
+          return s3Calls.deleteMatchingPrefix('handel-'+accountConfig.region+'-'+accountConfig.account_id,serviceContext.appName+'/'+serviceContext.environmentName+'/'+serviceContext.serviceName);
+          return;
+        })
+        .then(() => {
             return new UnDeployContext(serviceContext);
         });
 }
+exports.unDeployCloudFormationStack = function(serviceContext,serviceType) {
```

Comment:
I would vote for removing this little method since you've already switched over the usages of "unDeployCloudFormationStack" to "unDeployService".